src/sentry/tasks/store.py

```diff
@@ -1,7 +1,6 @@
 import logging
```

Comment:
From this file you will have to temporarily re-export all symbolication tasks as there can be in-flight queue items for which the worker will attempt to import from this module.

```suggestion
import logging
from sentry.tasks.symbolication import symbolicate_event, ...
```

Reply:
actually add yourself to https://github.com/getsentry/sentry/blob/a2212cd3f2cae4411c7c06bcbcecf03ba7bac2d7/src/sentry/conf/server.py#L543 instead